CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 - Update Flutter constraint
 - Make return value of toColorScheme extension method non-nullable ([#55](https://github.com/material-foundation/material-dynamic-color-flutter/pull/55))
 
+## 1.5.5
```

Comment:
Can you remove combine these this and the section above under the #1.5.5 heading and remove the NEXT heading?